multilabel_classification/pytorch_network_training.ipynb

```diff
@@ -0,0 +1,1064 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import timm\n",
+    "from torch import nn\n",
+    "from timm.data.loader import create_loader\n",
+    "import torch.utils.data as data\n",
+    "import torch\n",
+    "import pandas as pd\n",
+    "\n",
+    "from sklearn.metrics import balanced_accuracy_score\n",
+    "\n",
+    "\n",
+    "from imp import reload\n",
+    "from collections import defaultdict\n",
+    "from types import SimpleNamespace\n",
+    "\n",
+    "\n",
+    "import numpy as np\n",
+    "from sklearn.metrics import accuracy_score\n",
+    "from collections import Counter\n",
+    "from timm.optim import create_optimizer\n",
+    "import pickle"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timm.data import (\n",
+    "    resolve_data_config,\n",
+    ")\n",
+    "from timm.models import create_model"
```

Comment:
let's move all imports to top of notebook

Reply:
done : https://github.com/cleanlab/examples/pull/27/commits/28f62b8528fdc7aec2fd87bca2d5da8e142b7b1e